### ISSUE TITLE
[Improvement] Batch up "Tagged in a photo" notifications into a single email

### ISSUE THREAD

**Issue by szech** [JustinWingChungHui/MyFamilyRoot#131]
At the moment, the user is notified every time they are tagged. This might be annoying if they receive a whole gallery's worth of notifications. 

Instead of real-time notification, maybe they could get one email on a time delay (say 20 mins) with a summary of the activities?  

Or maybe you would want the first email to be instantaneous, but subsequent ones to be batched. 
Then the counter would have to be reset every 24 hours.

**Comment by JustinWingChungHui** [comment]
Yeah, I was thinking of something like this.  The immediacy is really cool in a live situation.  So I think you're right about the first one being instantaneous.

The time delay is difficult on PythonAnywhere as it only supports hourly tasks.  So we could batch the rest up within hourly buckets.

**Comment by JustinWingChungHui** [comment]
So we now have a message queue.  So could schedule emails better
